packages/suite-desktop-api/src/renderer.ts

```diff
@@ -0,0 +1,15 @@
+import { DesktopApi } from './api';
+import { factory } from './factory';
+
+export const getDesktopApi = () => {
```

Comment:
Should it be exported? Is it supposed to be used somewhere else instead of `desktopApi`?

Reply:
`renderer.js` is a browser replacement of `main.js`. it is exported in main.js (and used in src-electron/preload with ipcRenderer argument) so to be compatible i think it should be also exported here